go/scripts/update-codegen.sh

```diff
@@ -31,7 +31,7 @@ echo ${CODEGEN_PKG}
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
 ${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
- github.com/PaddlePaddle/cloud/go/pkg/client github.com/PaddlePaddle/cloud/go/pkg/apis \
+ github.com/PaddlePaddle/cloud/go/client github.com/PaddlePaddle/cloud/go/apis \
```

Comment:
I see no `go/pkg/client` directory in current develop branch, is this really needed? Or can you add this when the package is committed.

Reply:
@typhoonzero dir `client` will be generated for go test

Reply:
And I think it is strange only to change `pkg/apis`.

Reply:
I see, is `generate-groups.sh "deepcopy,client,informer,lister"` generating the template code for later development?

Reply:
Yes, if the structs of types change, the tools (client, deepcopy and so on) needs to be re-generated.